modules/nf-neuro/registration/synthregistration/main.nf

```diff
@@ -52,12 +51,12 @@ process REGISTRATION_SYNTHREGISTRATION {
     mri_synthmorph -h
 
     touch ${prefix}__output_warped.nii.gz
-    touch ${prefix}__affine_warp.lta
     touch ${prefix}__deform_warp.nii.gz
+    touch ${prefix}__affine_warp.lta
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        Freesurfer: 7.4
+        Freesurfer: 7.4.1
```

Comment:
why do you have different soft and versions here ? Generic way of getting the version also.

Reply:
Not for synthmorph yet, no mechanism in the container sadly. I aligned both versions for now